### PR TITLE
[S] Setup dummy app (#993)

### PR DESCRIPTION
**Pull request by SebastianPlace**
Closes https://github.com/repositive/report-issues/issues/993

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,11 @@
   },
   "dependencies": {
     "ember-cli-babel": "^6.4.1",
-    "ember-cli-sass": "^6.2.0",
-    "ember-cli-htmlbars": "^1.1.1"
+    "ember-cli-htmlbars": "^1.1.1",
+    "ember-cli-sass": "^6.2.0"
   },
   "devDependencies": {
+    "@repositive/liz-skin": "^1.1.0",
```
Comment (SebastianPlace) [inline package.json]: This won't be bundled with the addon.

File: tests/dummy/app/components/buttons-wrapper/template.hbs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      size="big"
+      variant="cancel"
+  }}
+</div>
+
+<h3 class="fc-secondary mb2">r-link-button</h3>
+<div class="mb2">
+  {{r-link-button
+      label="Link Button"
+      href="http://hackertyper.net/"
```
Comment (tombyers) [inline tests/dummy/app/components/buttons-wrapper/template.hbs:74]: 😂

**Review by tombyers — DISMISSED** [review]
This is pretty cool Seb!

**Comment by tombyers** [comment]
Also. @SebastianPlace Out of interest. How long did this take?

**Comment by SebastianPlace** [comment]
@tombyers roughly 2-3 hours. Everything just seemed to work, no issues.

**Comment by Istar-Eldritch** [comment]
I think we can do this directly from nyc in the package json. Ill check it tonight. Looks on to me for now

**Review by tombyers — APPROVED** [review]

**Comment by SebastianPlace** [comment]
@Istar-Eldritch I was hoping I could put it in `package.json` but couldn't find anything in the [docs](https://docs.codecov.io/v4.3.6/docs/codecov-yaml) about it.

**Pull request merged**